tests/clients/test_securitydata.py

```diff
@@ -1321,6 +1336,166 @@ def test_stream_file_by_md5_when_find_file_version_returns_204_status_code_raise
 
         assert e.value.args[0] == PDS_EXCEPTION_MESSAGE.format("mdhash")
 
+    def test_stream_file_by_md5_when_has_exact_match_calls_get_token_with_expected_params_and_streams_successfully(
+        self,
+        security_service,
+        file_event_service,
+        preservation_data_service,
+        saved_search_service,
+        storage_service_factory,
+        file_event_search,
+        file_download,
+        mocker,
+    ):
+        file_version_list = py42_response(mocker, XFC_EXACT_FILE_VERSIONS)
```

Comment:
A fixture (or fixtures) and/or method for the setup of this test and the next would be nice. It'd help us manage testing states in a global way, and a familiar way if uses fixtures.